sdk/nodejs/dagger/client.ts

```diff
@@ -1,62 +1,98 @@
-import axios, { AxiosInstance } from "axios";
-import { buildAxiosFetch } from "@lifeomic/axios-fetch";
-import { GraphQLClient } from "graphql-request";
+import axios from 'axios';
+import { buildAxiosFetch } from '@lifeomic/axios-fetch';
+import { GraphQLClient } from 'graphql-request';
+
+import { Response } from 'node-fetch';
+import { ServerProcess } from './connect.js';
 
-import { Response } from "node-fetch";
 // @ts-expect-error node-fetch doesn't exactly match the Response object, but close enough.
 global.Response = Response;
 
-export const client = new GraphQLClient("http://fake.invalid/query", {
-  fetch: buildAxiosFetch(
-    axios.create({
-      socketPath: "/dagger.sock",
-      timeout: 3600e3,
-    })
-  ),
+export const client = new GraphQLClient('http://fake.invalid/query', {
+	fetch: buildAxiosFetch(
+		axios.create({
+			socketPath: '/dagger.sock',
+			timeout: 3600e3,
+		})
+	),
```

Comment:
I could not find why but the client does not work when it's using socket...
That only work when I use `GraphQLClient` that hits `http://localhost${port}`.

Reply:
Can you create an issue for that please @TomChv  🙏 ? Let's move forward on this PR